examples/preact-kitchen-sink/jest.config.js

```diff
@@ -0,0 +1,5 @@
+const config = require('../../jest.config');
```

Comment:
this file is needed only if you have tests (mostly for storyshots) in the example app

Reply:
Thanks, I now added storyshots correctly to the example app.